docs/security/principles/03-no-security-on-fully-compromised.mdx

```diff
@@ -0,0 +1,21 @@
+# P03 - No security on fully compromised systems
```

Comment:
**point-of-interest:** this was previously called `No guarantees for unlocked vaults on fully compromised devices`. I changed it to make it more consistent with the other principles, but "no security" might seems a bit harsh

Reply:
Related to the confluence point; I think we might need to add "no security on fully compromised bitwarden contexts"